dwave/system/testing.py

```diff
@@ -65,21 +62,30 @@ class MockDWaveSampler(dimod.Sampler, dimod.Structured):
             Supported options are 'chimera', 'pegasus' or 'zephyr'.
             The default is 'chimera' when the value is not specified as part 
             of the from the ``properties`` argument.
-            
+
         topology_shape (string, optional):
             A list of three numbers [m,n,t] for Chimera, defaulted as [4,4,4]. 
             A list of one number [m] for Pegasus, defaulted as [3].
             A list of two numbers [m,t] for Zephyr, defaulted as [2,4].
             The default above apply only when the value is not 
             specified as part of the from the ``properties`` argument.
-            
+
         parameter_warnings (bool, optional, default=True):
             The MockSampler is adaptive with respect to ``num_reads``,
             ``answer_mode`` and ``max_answers`` and ``label`` 
             parameters. By default ``initial_state`` can also be mocked, if
             dwave-greedy is installed. All other parameters are ignored and a 
             warning will be raised by default.
 
+        exact_solver_cutoff (int, optional, default=:attr:`EXACT_SOLVER_CUTOFF_DEFAULT`):
+            For problems smaller or equal in size to ``exact_solver_cutoff`` the
+            first sample in any sampleset returned by the class sampling routines
+            is replaced by a reproducible ground state (determined exactly with
+            :class:`~dimod.ExactSolver`). Only small cut offs should be used since
+            time resources scale exponentially in system size.
+            Set ``exact_solver_cutoff`` to zero to disable exact ground state
+            calculation.
```

Comment:
- For problems smaller or equal in size to ``exact_solver_cutoff`` --> comma
- "the class sampling routines" --> can this be stated plainly
- reproducible ground state  -->  this is clear when you know what this PR want to address but will puzzle users. Perhaps something like "by a ground state found through a brute-force solution method (making it reproduceable for testing)"
- time resources scale exponentially in system size --> perhaps "solution time increases exponentially with problem size"